tests/serialization/test_arrow_compat.py

```diff
@@ -28,7 +36,7 @@ def reference_store():
     return get_store_from_url("hfs://{}".format(path))
 
 
-def test_arrow_compat(arrow_version, reference_store, mocker):
+def test_arrow_compat(arrow_version, reference_store, mocker, forward_compatibility):
```

Comment:
I thought arrow was going to fix forward compatibility. Is this still broken?

Reply:
Not sure, will test later.